e2e/e2e_test.go

```diff
@@ -375,37 +398,50 @@ func Test_EVM_Hedera_Token(t *testing.T) {
 		setupEnv.Clients.Hedera.GetOperatorAccountID().String(),
 		status.Completed,
 		evm.Signer.Address(),
-		entity.NanoTime{Time: blockTimestamp})
+		entity.NanoTime{Time: blockTimestamp},
+	)
 	// and:
-	expectedFeeRecord := util.PrepareExpectedFeeRecord(transactionID, scheduleID, fee, expectedId)
+	expectedFeeRecord := expected.FeeRecord(transactionID, scheduleID, fee, expectedId)
 
 	// 8. Wait for validators to update DB state after Scheduled TX is mined
-	time.Sleep(30 * time.Second)
```

Comment:
I think we will need a more stable fix, as we already sync we could have a re-try in case of timeout.